packages/components/src/templates/next/components/internal/Siderail/SiderailList.tsx

```diff
@@ -0,0 +1,87 @@
+import type { PropsWithChildren } from "react"
+import {
+  Disclosure,
+  DisclosureButton,
+  DisclosurePanel,
+} from "@headlessui/react"
+import { BiMinus, BiPlus } from "react-icons/bi"
+import { twMerge } from "tailwind-merge"
+import { tv } from "tailwind-variants"
+
+import type { ItemWithChild } from "./types"
+import { SiderailLabel } from "./SiderailLabel"
+
+export interface SiderailListProps {
+  item: ItemWithChild
+  className?: string
+}
+
+const createSiderailListStyles = tv({
+  slots: {
+    header:
+      "flex w-full items-start justify-between gap-2 text-base-content-strong transition",
+    label: "prose-headline-base-semibold px-3 py-4",
+    childHeader: "flex w-full items-center gap-1.5",
+    childLabel: "py-2 hover:text-brand-interaction",
+    container: "",
+    expandButton: "py-4",
+    expandIcon: "h-6 w-6 transition hover:text-brand-interaction",
```

Comment:
issue: hahaha I'm sorry but it is not aligned

<img width="282" alt="Screenshot 2024-08-01 at 14 52 27" src="https://github.com/user-attachments/assets/2e0dc949-4700-48b6-b1ae-e905d26253f2">
<img width="277" alt="Screenshot 2024-08-01 at 14 53 06" src="https://github.com/user-attachments/assets/cc3d264b-e445-41ae-bce3-96b3d64ecd30">
<img width="288" alt="Screenshot 2024-08-01 at 14 54 53" src="https://github.com/user-attachments/assets/cfbb655d-dd5e-43ae-a83d-66fbc807a326">


But okay lah at least the arrows themselves are aligned, just the plus minus signs

<img width="286" alt="Screenshot 2024-08-01 at 14 53 53" src="https://github.com/user-attachments/assets/a9b66bc2-e875-41ef-b6c4-aaa801e020d9">